faust/tables/recovery.py

```diff
@@ -545,22 +549,17 @@ def _estimated_active_remaining_secs(self, remaining: float) -> Optional[float]:
         else:
             return None
 
-    async def _wait(self, coro: WaitArgT) -> None:
-        wait_result = await self.wait_first(
-            coro,
-            self.signal_recovery_reset,
-            self.signal_recovery_start,
-        )
+    async def _wait(self, coro: WaitArgT, timeout=None) -> None:
```

Comment:
let's use the annotation `Seconds` for timeout (`from mode.utils.times import` )

Reply:
using int